tools/eggnog_mapper/eggnog_mapper.xml

```diff
@@ -0,0 +1,491 @@
+<tool id="eggnog_mapper" name="eggNOG Mapper" version="0.2.0">
+    <description>functional sequence annotation by orthology</description>
+    <requirements>
+        <requirement type="package" version="1.0.1">eggnog-mapper</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <version_command>emapper.py --version</version_command>
+    <command><![CDATA[
+        emapper.py
+        --data_dir '$eggnog_data.fields.data_dir'
+        -m $db.mode 
+        $translate
+        #if ($db.mode == 'hmmer'):
+            #if ($test):
+                --database=$db.database.fields.data_dir
+            #else
+                --database=$db.database
+            #end if
+            #if $db.hmm_options.hmm_settings == 'specified':
+                --hmm_maxhits=$db.hmm_options.hmm_maxhits
+                --hmm_evalue=$db.hmm_options.hmm_evalue
+                --hmm_score=$db.hmm_options.hmm_score
+                --hmm_maxseqlen=$db.hmm_options.hmm_maxseqlen
+                #if str($db.hmm_options.hmm_qcov):
+                    --hmm_qcov=$db.hmm_options.hmm_qcov
+                #end if
+                --Z=$db.hmm_options.Z
+            #end if
+        #elif ($db.mode == 'diamond'):
+            ## #if $db.sensitivity == "1":
+            ##     --sensitive
+            ## #else if $db.sensitivity == "2":
+            ##     --more-sensitive
+            ## #end if
+            --matrix '$db.matrix_gapcosts.matrix'
+            $db.matrix_gapcosts.gap_costs
+        #end if
+        #if $annotation_options.tax_scope:
+            --tax_scope=$annotation_options.tax_scope
+        #end if
+        #if $annotation_options.target_orthologs:
+            --target_orthologs=$annotation_options.target_orthologs
+        #end if
+        #if $annotation_options.go_evidence:
+            --go_evidence=$annotation_options.go_evidence
+        #end if
+        #if $seed_ortholog_options.seed_ortholog_evalue:
+            --seed_ortholog_evalue=$seed_ortholog_options.seed_ortholog_evalue 
+        #end if
+        #if str($seed_ortholog_options.seed_ortholog_score):
+            --seed_ortholog_score=$seed_ortholog_options.seed_ortholog_score 
+        #end if
+        $output_options.no_file_comments
+        $output_options.no_annot
+        $output_options.report_orthologs
+        --output='results'
+        -i '${input}'
+    ]]></command>
+        <inputs>
+            <!-- used for testing -->
+            <param name="test" type="hidden" value="false"/>
```

Comment:
I don't understand why this is needed?

Reply:
good catch! this was left over from my earlier attempts to make testing work